plugins/datadog-standalone/datadog-standalone.go

```diff
@@ -134,6 +135,7 @@ func postMetricsToDatadog(datadogURL string, metrics []producers.MetricsMessage)
 	result := DDResult{}
 	err = json.Unmarshal(body, &result)
 	if err != nil {
+		log.Warnf("Unreadable DataDog output: %s", body)
```

Comment:
I think it would be good to also include the err here in the log message.

```
log.Warnf("Could not unmarshal datadog response %s: %s", body, err)
```